2023-03_sprint/sprint_parties/contributors.md

```diff
@@ -111,3 +111,16 @@ Brooklyn, New York
 [{fab}`github`](https://github.com/vsaitejaswie)
 :::
 ::::
```

Comment:
Also the grid item needs to be _inside_ the grid directive. That is, within the `::::{grid}` opening and the `::::` closing in this line (note the difference between this 4 colon fences and the innner ones of 3 colons only for each grid item)

Reply:
You can see in the preview how your info is outside the grid: https://pymc-data-umbrella--224.org.readthedocs.build/en/224/2023-03_sprint/sprint_parties/contributors.html

Reply:
@OriolAbril I have updated the grid item positioning.